packages/next/next-server/server/incremental-cache.ts

```diff
@@ -39,13 +44,17 @@ export class IncrementalCache {
     pagesDir,
     flushToDisk,
     locales,
+    storageProvider,
+    lruCacheProvider,
   }: {
     dev: boolean
     max?: number
     distDir: string
     pagesDir: string
     flushToDisk?: boolean
     locales?: string[]
+    storageProvider?: StorageProfiderInterface
```

Comment:
Instead of passing this in here, could it be an implementation detail of the `lruCacheProvider`?

In other words, could the default `lruCacheProvider` be built of both an in-memory LRUCache and a FilesystemCache fallback?